opentelemetry-instrumentation/src/opentelemetry/instrumentation/_semconv.py

```diff
@@ -85,13 +96,12 @@
     SpanAttributes.HTTP_SCHEME,
     SpanAttributes.HTTP_FLAVOR,
     SpanAttributes.HTTP_SERVER_NAME,
-    SpanAttributes.NET_HOST_NAME,
```

Comment:
Can we have a changelog saying we are removing this? I understand it was something wrong introduced in the previous PR